frontend/src/features/properties/list/components/DirtyRowsTracker.tsx

```diff
@@ -0,0 +1,33 @@
+import { getIn, useFormikContext } from 'formik';
+import isEmpty from 'lodash/isEmpty';
+import { useEffect } from 'react';
+
+interface IChangedRow {
+  rowId: number;
+  assessedLand?: boolean;
+  assessedBuilding?: boolean;
+  netBook?: boolean;
+  market?: boolean;
+}
+
+/**
+ *  Component to track edited rows in the formik table
+ * @param param0 {setDirtyRows: event listener for changed rows}
+ */
+export const DirtyRowsTracker: React.FC<{ setDirtyRows: (changes: IChangedRow[]) => void }> = ({
```

Comment:
Are you using this? If not we can just delete this.

Reply:
No I am not. Happy to delete. I didn't in case we needed but we have the history in git. Will remove